playbooks/responses/blocked-needs-repro.md

```diff
@@ -2,24 +2,25 @@
 
 Thank you for taking the time to report this issue and helping to make Electron better.
 
-Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself?
+Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) to make a stand-alone app that reproduces this issue?
 
 Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.
 
-I'm adding the `blocked/needs-repro` label for this reason. After you've responded, please link to the test case and @ me in a followup comment.
+I'm adding the `blocked/needs-repro` label for this reason. After you make a test case (a [gist](https://gist.github.com) is preferred since gists can be loaded directly into [Electron Fiddle](https://www.electronjs.org/fiddle) for testing), please link to it in a followup comment.
```

Comment:
I'm not sure if linking to GitHub Gists is useful here - we probably want to emphasize the use of Fiddle to save the Gist, as opposed to them trying to do it by hand.

Reply:
Hmm you may be right. I intentionally avoided that step because installing Fiddle could be seen as burdensome to a bug reporter; but then again, you could make the same argument against gists. And if we recommend Fiddle we'll get gist support built-in, while the reverse is not the case.

@codebytere commented above that endorsing electron-quick-start _and_ gists might be too confusing as well. Maybe Fiddle is the answer to all of this, since we would just recommend fiddle and that tool automatically points people in the right direction.

Reply:
Update: I've revised this again in a26ffd95ff68d1725a6655b2fad81fe3189c0979 based on feedback from @codebyte and @malept